### PR TITLE
Adding notable

### PR DESCRIPTION
**Pull request by jasondew**
/cc @mlettini @katierose4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,17 +7,17 @@
...
 </head>
+
 <body>
   <div id="container">
-    
     <header>
       <h1>Open Source @ <a href="http://www.getharvest.com" target="_blank">Harvest</a></h1>
       <p>
-        Want to work on projects like these? <a href="http://www.getharvest.com/careers" target="_blank">We’re hiring</a>
+        Want to work on projects like these? <a href="http://www.getharvest.com/careers" target="_blank">Contact us!</a>
```
Comment (mlettini) [inline index.html]: Please change the url to the contact page too

File: index.html [hunk 2]
```diff
@@ -33,16 +33,10 @@
...
-      </a>
-
-      <a href="https://github.com/harvesthq/simple_xlsx_writer" class="project" target="_blank">
-        <h2>simple_xlsx_writer</h2>
-        <p>Ruby</p>
-        <p>This is a simple, no-fuss generator for OpenXML (aka XLSX) files. No formatting or styles included, just few basic datatypes.</p>
+      <a href="https://github.com/harvesthq/notable" class="project" target="_blank">
+        <h2>notable</h2>
+        <p>Go</p>
+        <p>Notable makes it easy to share notable announcements with your team via Slack notifications and email digests</p>
```
Comment (mlettini) [inline index.html]: All the other sentences end in a period, this should too

**Comment by mlettini** [comment]
2 comments

**Comment by jasondew** [comment]
@mlettini done, also updated all links to be `https`

**Comment by mlettini** [comment]
+1

**Pull request merged**